nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
+tags:
+ - upgrade
+ - migration
+related:
+ - samples/scaleout/distributor-upgrade
+---
+
+The [distributor](/nservicebus/scalability-and-ha/distributor) has been deprecated and replaced by [sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution.md) with NServiceBus Version 6.
+
+## Client-side distribution
+
+With sender-side distribution, senders are aware of scaled out receivers, allowing senders to distribute messages directly to the receivers instead of using an intermediary distributor.
+
+
+## Upgrade scenario
+
+This process aims to allow upgrade in production environments without missing events and minimal downtime.
+
+WARNING: To avoid duplicate events, the Distributor node needs to be upgraded to a worker node using NServiceBus 6 (it can be decommissioned after all workers are upgraded to Version 6).
```

Comment:
And honestly I'm not sure I know what this text means, especially since it's not clear if you're talking about starting from a Master+Worker or Distributor Only configuration.